arviz/tests/external_tests/test_data_pyjags.py

```diff
@@ -16,6 +20,41 @@
 PYJAGS_PRIOR_DICT = {"b": np.random.randn(3, 10, 3), "int": np.random.randn(1, 10, 3)}
 
 
+EIGHT_SCHOOL_DATA = {
+    "J": 8,
+    "y": np.array([28.0, 8.0, -3.0, 7.0, -1.0, 1.0, 18.0, 12.0]),
+    "sigma": np.array([15.0, 10.0, 16.0, 11.0, 9.0, 11.0, 10.0, 18.0]),
+}
```

Comment:
this data is already available as a fixture: https://github.com/arviz-devs/arviz/blob/master/arviz/tests/helpers.py#L21 it should be possible to import and add as function argument to use it from helpers.

Reply:
Ok, I changed that.